.github/workflows/ci.yml

```diff
@@ -22,21 +22,21 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.12.10, 2.13.1]
-        java: [temurin@8]
+        scala: [2.13.1, 3.3.1]
```

Comment:
Why 2.13.1? The latest 2.13.x release is 2.13.12
Is this a typo, like everywhere in this PR?

Reply:
[2.13.1 was the previously set version of Scala](https://github.com/leanovate/play-mockws/blob/88a0c36e98bab7c810b292c5f5980c221e0dbc9d/build.sbt#L9) that this project was using, I avoided bumping anything more than I needed to get this project to work with Play 2.9 and Scala 3